nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -163,7 +163,11 @@
 - module-name: 'arcade' # checksum: 3f9f0b7d
   data-files:
     dirs:
-      - 'resources'
+      - 'resources/system/'
```

Comment:
That should actually be wrong, to have a trailing slash, pretty sure the checker catches it even.